### PR TITLE
Added Discriminator #15

### PR DESCRIPTION
**Pull request by captain-pool**
Added VGG 128 Discriminator 
Issue: #15

### CODE REVIEW

File: E2_ESRGAN/lib/model.py [hunk 1]
```diff
@@ -46,3 +45,29 @@ def call(self, input_):
...
     return out
+
+
+class VGGArch(tf.keras.Model):
+  def __init__(self, output_shape=[1], num_features=64, use_bias=True):
+
+    super(Discriminator, self).__init__()
+    self.conv = lambda n, s, x: tf.keras.layers.Conv2D(
+        n, kernel_size=[3, 3], strides=[s, s], use_bias=use_bias)(x)
+    self.nf = num_features
```
Comment (srjoglekar246) [inline E2_ESRGAN/lib/model.py]: `self.num_features`. Names like `nf` get quickly hard to disambiguate once the code gets large enough :-)

File: E2_ESRGAN/lib/model.py [hunk 2]
```diff
@@ -46,3 +45,29 @@ def call(self, input_):
     feature = self.lrelu(self.conv_last_1(feature))
     out = self.conv_last_2(feature)
     return out
+
+
+class VGGArch(tf.keras.Model):
```
Comment (srjoglekar246) [inline E2_ESRGAN/lib/model.py]: Lets add docstrings, here and to the previous class.

Comment (captain-pool) [reply E2_ESRGAN/lib/model.py]: I've addressed adding docstrings as one of the tasks, #23, Will be adding them when I address this PR. What do you suggest?

File: E2_ESRGAN/lib/model.py (hunk 1 above)
Comment (captain-pool) [reply E2_ESRGAN/lib/model.py]: Fixed it :)

File: E2_ESRGAN/lib/model.py (hunk 2 above)
Comment (srjoglekar246) [reply E2_ESRGAN/lib/model.py]: SG.

**Pull request merged**